lib/ui/views/withdraw/withdraw_view.dart

```diff
@@ -32,6 +39,45 @@ class _WithdrawViewState extends State<WithdrawView> {
 
     final walletBTC = Provider.of<BtcRealTimeWallet>(context);
     final walletEth = Provider.of<EthRealTimeWallet>(context);
+    final walletUsdt = Provider.of<UsdtRealTimeWallet>(context);
+    final walletUni = Provider.of<UniRealTimeWallet>(context);
+    final walletBat = Provider.of<BatRealTimeWallet>(context);
+
+    walletBalanceText() {
+      String cryptoBalance = '';
+      if (types == CryptoTypes.bitcoin) {
+        cryptoBalance =
+            (walletBTC.balance * (1 / pow(10, 8))).toStringAsFixed(6) +
+                ' $cListVal';
+      } else if (types == CryptoTypes.ethereum) {
+        cryptoBalance =
+            (walletEth.balance * (1 / pow(10, TokenDecimals.ethTokenDecimals)))
```

Comment:
You can extract the logic of calculating the balance to the Wallet class so you don't repeat the logic